### PR TITLE
Create configtips.yml

### PR DESCRIPTION
**Pull request by alexvsav**
I hope our project qualifies for up for grabs!
Happy Hacktoberfest 🦾

### CODE REVIEW

**Review by alexvsav — COMMENTED** [review]
Updated to fit guidelines

File: _data/projects/configtips.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - github
+  - vite
+  - aws
+  - circleci
+  - nextjs
+  - nginx
+  - packagejson
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/CircleCI-Public/Config.Tips/issues/
```
Comment (shiftkey) [inline _data/projects/configtips.yml:17]: 🤔 this should link to a list of issues in the repository that are ideal for contributors. The only issue with this label is https://github.com/CircleCI-Public/Config.Tips/issues/63 which looks like a greeting.

**Review by shiftkey — COMMENTED** [review]
In it's current form this isn't an ideal fit for Up for Grabs

**Comment by shiftkey** [comment]
Closing this out as I think https://github.com/CircleCI-Public/Config.Tips/issues/63 is very broad, and not aligned with how this project is setup

**Pull request closed without merging**